stdlib/arrayLabels.mli

```diff
@@ -339,6 +339,19 @@ val stable_sort : cmp:('a -> 'a -> int) -> 'a array -> unit
    than the current implementation of {!sort}.
 *)
 
+val stable_sort_segment : ('a -> 'a -> int) -> 'a array -> int -> int -> unit
```

Comment:
Sorry if this is a dumb question, but shouldn't we use labels for `ofs` and `len` since we're in the `Labels` version of this module?